tests/Manifest1.jad

```diff
@@ -3,6 +3,6 @@ MIDlet-Jar-Size: 2436
 MIDlet-Jar-URL: tests.jar
 MIDlet-Name: AMIDlet
 MIDlet-Vendor: MIDlet Suite Vendor
-MIDlet-Version: 1.0.0
+MIDlet-Version: 1

```

Comment:
Well, it was probably done on purpose to make RunTests work with class names both with points and with slashes.